play/symphony-conductor/index.html

```diff
@@ -236,6 +237,24 @@ <h3 id="label10">Video Tutorial</h3> <h3 id="label11">Examples</h3>
     <script src="../js/lzma.js"></script>
 
     <script src="js/app.js"></script>
+
+
+    <div id="LoadingGif" style="display:none"> <img src="img/LoadingGif.gif"></div>
+    <div id="LoadingBack" style="display:none"> <img src="img/LoadingBack2.png" width="100%" height="100%"></div>
+
+    <script>
```

Comment:
You don't have to put this script inline, you could put it in app.js

Reply:
I transferred put this function into app.js